tests/runtime/http_server/test_http_server.py

```diff
@@ -486,20 +486,6 @@ def test_inference_other_task(other_task_model_id, client):
     assert json_response["farewell"] == "goodbye: world 42 times"
 
 
-def test_inference_no_aborter(other_task_model_id, client):
```

Comment:
Why is this removed?

Reply:
There's another test that does the same thing: `test_inference_other_task`
🤷